src/test/resources/tls/client-trusted-root-ca/.gitignore

```diff
@@ -0,0 +1 @@
+*.old
```

Comment:
It's better to use the main project .gitignore file with something like : 

**/*.old

instead of more .gitignore files here.